conf/ref_databases.config

```diff
@@ -9,6 +9,13 @@
 
 params {
     dada_ref_databases {
+        'midori2-co1=gb250' {
+            title = "MIDORI2 - CO1 Taxonomy Database - Release GB250"
+            file = [ "http://reference-midori.info/download/Databases/GenBank250/DADA2/uniq/MIDORI2_UNIQ_NUC_GB250_CO1_DADA2.fasta.gz", "http://reference-midori.info/download/Databases/GenBank250/DADA2_sp/uniq/MIDORI2_UNIQ_NUC_SP_GB250_CO1_DADA2.fasta.gz" ]
+            citation = "Machida RJ, Leray M, Ho SL, Knowlton N. Metazoan mitochondrial gene sequence reference datasets for taxonomic assignment of environmental samples. Sci Data. 2017 Mar 14;4:170027. doi: 10.1038/sdata.2017.27. PMID: 28291235; PMCID: PMC5349245."
+            fmtscript = "taxref_reformat_midori2.sh"
+            dbversion = 'midori2-co1=gb250'
+        }
```

Comment:
We should have a generic without version number, right? We do for the other databases.

Reply:
done!